drydock/templates/drydock/k8s/ingress/lms.yml

```diff
@@ -0,0 +1,74 @@
+---
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: ingress-lms
+  namespace: {{ K8S_NAMESPACE }}
+  annotations:
+    kubernetes.io/ingress.class: nginx
+    {%- if DRYDOCK_AUTO_TLS and not DRYDOCK_CUSTOM_CERTS %}
+    cert-manager.io/issuer: letsencrypt
+    {%- endif %}
+spec:
+  rules:
+  - host: {{ LMS_HOST }}
+    http:
+      paths:
+      - pathType: Prefix
+        path: "/learning"
+        backend:
+          service:
+            name: {% if DRYDOCK_BYPASS_CADDY -%}mfe{% else -%}caddy{% endif %}
+            port:
+              number: {% if DRYDOCK_BYPASS_CADDY -%}9001{% else -%}80{% endif %}
+      - pathType: Prefix
+        path: "/"
+        backend:
+          service:
+            name: {% if DRYDOCK_BYPASS_CADDY -%}lms{% else -%}caddy{% endif %}
+            port:
+              number: {% if DRYDOCK_BYPASS_CADDY -%}8000{% else -%}80{% endif %}
+  {{ patch("drydock-lms-ingress-rules")|indent(2) }}
```

Comment:
I wonder if we better apply a patch via Kustomize to this resource when we need extra rules instead of supporting yet another tutor patch.

Reply:
We can set this in kustomize-strategic-merge adding the new hosts, the good thing with that is we can really set `paths` to lms host instead of create new hosts. Any thoughts @MoisesGSalas ?

Reply:
@Henrrypg as we discussed, we should have a variable like `DRYDOCK_INGRESS_LMS_EXTRA_HOSTS` and iterate it here to define the paths for all LMS hosts. I wouldn't support custom certs per tenant for now